pages/index.js

```diff
@@ -1,14 +1,27 @@
 import Head from "next/head";
 import styles from "styles/Home.module.css";
 import Layout from "components/Layout"
+import HomeProducts from "components/HomeProducts"
 
-export default function Home() {
+export default function Home(props) {
   return (
     <Layout>
       <Head>
         <title>Ecommerce Coffee</title>
       </Head>
       <div className={styles.container}></div>
+      <HomeProducts coffees={props.firstNine} itemsLeft={props.itemsLeft} />
     </Layout>
   );
 }
+
+export async function getStaticProps() {
+  const res = await fetch("http://localhost:3000/api/hello")
```

Comment:
ohhh I see you are prefetching the data before the render. Nice

Reply:
I still have to change the API call using limits? :sweat_smile:

Reply:
Yes, that would be better, also take in consideration that item left, it would be obsolete if a new product is added and the user has not refresh the page.

So we need to handle if the user can get more products in the backend on every request.